client/src/components/TodoCardList/Item/index.js

```diff
@@ -1,17 +1,29 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import { Card, CardBody, Input } from 'mdbreact'
+import { Card, CardBody } from 'mdbreact'
+
+import styles from 'src/components/TodoCardList/Item/styles.css'
 
 const Item = props => (
   <div className="pt-1">
     <Card>
-      <CardBody className="text-left">
-        <Input
-          checked={props.completedAt}
-          onClick={props.onChange}
-          type="checkbox"
-          label={props.description}
-        />
+      <CardBody>
+        <div className="row">
+          <div className="col-sm-2">
+            <input
+              type="checkbox"
+              value=""
+              onChange={props.onChange}
+              checked={props.completedAt}
+            />
+          </div>
+          <span className="col-sm-8 text-left">
+            {props.description}
+          </span>
+          <a className="col-sm-2" href="http://localhost:3000/" styles={styles}>
```

Comment:
@datarockets/cuties thoughts about having a file called `server.js` in the root of react app in order to keep all routes there?

So, we'll have:
```js
import server from 'src/server'

...

<a className="col-sm-2" href=`${server.baseUrl}` styles={styles}>
```